articles/proofs-arguments-and-zero-knowledge/chapter_3.md

```diff
@@ -0,0 +1,62 @@
+# Chapter3 - Definitions and Technical Preliminaries
+### 3.1 Interactive Proofs
+Prover (**P**) claims that he has a value y = f(x). To verify it, verifier (**V**) exchanges messages with **P** back and forth.  At the end of this protocol, **V** must output either 0 or 1, with 1 indicating that the verifiers accept the prover’s claim that y = f(x) and 0 indicating that the verifier rejects the claim.
+The entire sequence of k messages $t := (m_1,m_2,...,m_k)$ exchanged by $P$ and $V$, along with the claimed answer y, is called a transcript.
+$Out(V,x,r,P) ∈ \{0,1\}$ the output of verifier $V$ on input $x$ when interacting with deterministic prover strategy P, with V’s internal randomness equal to r. 
+.An interactive proof system (V,P) is said to have completeness error $δ_c$ and soundness error $δ_s$ : 
+- Completeness error $δ_c$ :        $\text{Pr}[\text{out}(V, x, r, P) = 1]$  $\ge$ 1 -  $δ_c$
+- Soundness error $δ_s$:              $\text{Pr}[\text{out}(V, x, r, P') = 1]$ $\le$  $δ_s$,   which $P'$ is prover strategy with value: $y \ne f(x)$(*statistical soundness*)
+An interactive proof system is valid if $δ_c$ , $δ_s$ ≤ 1/3.
+
+### 3.2 Argument Systems
+**Definition 3.2:** An argument system for a function $f$ is an interactive proof for $f$ in which the soundness condition is only required to hold against prover strategies that run in polynomial time (*computational soundness*).
+
+### Robustness of Definitions and the Power of Interaction
+- Any IP for a function f with $δ_c$ ≤ 1/3 can be transformed into perfect completeness ($δ_c$ = 0), with a polynomial blowup in the verifier’s costs.
```

Comment:
Replace normal "IP" with the one that is "clickable", same with "NP". Remove the last line